content/Reducing_Power_Consumption.md

```diff
@@ -5,18 +5,19 @@ batteries.
 # Using the HALT Instruction
 
 The HALT instruction should be used whenever possible to reduce power
-consumption & extend the life of the batteries. This command stops the
+consumption and extend battery life. This instruction halts the
 system clock, reducing the power consumption of both the CPU and ROM.
 
-The CPU will remain stopped until an interrupt occurs at which point the
-interrupt is serviced and then the instruction immediately following the
-HALT is executed.
+The CPU will remain halted until an interrupt occurs, at which point the
+interrupt is serviced (if IME is enabled and the interrupt is enabled in IE)
+and then execution continues at the instruction immediately following the
+HALT.
 
 Depending on how much CPU time is required by a game, the HALT
-instruction can extend battery life anywhere from 5 to 50% or possibly
+instruction can extend battery life anywhere from 5% to 50% or possibly
 more.
 
-When waiting for a vblank event, this would be a BAD example:
+When waiting for a v-blank event, this would be a BAD example:
```

Comment:
This has not been fully decided yet (#94), but the consensus so far seems to be `VBlank` rather than `V-Blank`, `v-blank`, etc.

You are free to weigh in on #94 if you disagree.

Reply:
See https://github.com/gbdev/pandocs/wiki/Document-Style#6-horizontal-blanking-interval-vertical-blanking-interval

Reply:
So, VBlank please.

Reply:
Fixed

Reply:
There's just one line 45.

Reply:
Fixed